curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/positive-and-negative-lookahead.md

```diff
@@ -70,22 +70,22 @@ Your regex should match `"abc123"`
 assert(pwRegex.test('abc123'));
 ```
 
-Your regex should not match `"1234"`
+Your regex should not match `"12345"`
 
 ```js
-assert(!pwRegex.test('1234'));
+assert(!pwRegex.test('12345'));
 ```
 
-Your regex should not match `"8pass99"`
+Your regex should match `"8pass99"`
```

Comment:
```suggestion
Your regex should not match `"8pass99"`
```

It should not start with a number.

Reply:
I changed the instructions to remove that requirement...
So, it **may** start with numbers, now.

Reply:
@moT01 Based on this comment, and the agreement: https://github.com/freeCodeCamp/freeCodeCamp/issues/35700#issuecomment-478314901

Reply:
My bad, misread it or something.